libs/libcap-ng/Makefile

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (C) 2020 Lucian Cristian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libcap-ng
+PKG_VERSION:=0.7.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://people.redhat.com/sgrubb/libcap-ng
+PKG_HASH:=a84ca7b4e0444283ed269b7a29f5b6187f647c82e2b876636b49b9a744f0ffbf
+
+PKG_MAINTAINER:=Lucian CRISTIAN <lucian.cristian@gmail.com>
+PKG_LICENSE:=GPL-2.0-or-later LGPL-2.1-or-later
+PKG_LICENSE_FILES:=License
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libcap-ng/Default
+  TITLE:=POSIX capabilities library
+  SECTION:=libs
+  CATEGORY:=Libraries
+  URL:=https://people.redhat.com/sgrubb/libcap-ng/index.html
+endef
+
+define Package/libcap-ng/description/Default
+  POSIX capabilities library
+endef
+
+define Package/libcap-ng
+  $(call Package/libcap-ng/Default)
+  TITLE += library
```

Comment:
I'm not sure if this is all right as this is the same as in `libcap` Makefile.

![image](https://user-images.githubusercontent.com/4096468/82151678-eedb8b00-985c-11ea-9f20-afa35467ba6f.png)

Reply:
is the same as is the -ng version, the bin files are different
capsh  getcap  getpcaps  setcap
vs
captest  filecap  netcap  pscap

Reply:
I mean that there is twice library in TITLE.

Reply:
Ah, I see, I copied it as the template